hashes/buildroot

```diff
@@ -1,28 +1,30 @@
+SHA512 (gcc-13.3.0.tar.xz) = ed5f2f4c6ed2c796fcf2c93707159e9dbd3ddb1ba063d549804dd68cdabbb6d550985ae1c8465ae9a336cfe29274a6eb0f42e21924360574ebd8e5d5c7c9a801
+# https://github.com/bminor/glibc/archive/5641780762723156b0d20a0b9f7df1d76831bab0.tar.gz
+SHA512 (glibc-2.40-18-g5641780762723156b0d20a0b9f7df1d76831bab0.tar.gz) = af334c81984ef685968ac400125542c0ac434756ff8da861e0cc7b09cba46bb681ff551da33fc35f4037f745b5284268f45df44e438fc77807062551aee82307
 # https://mirrors.kernel.org/gnu/gmp/gmp-6.3.0.tar.xz
 SHA512 (gmp-6.3.0.tar.xz) = e85a0dab5195889948a3462189f0e0598d331d3457612e2d3350799dba2e244316d256f8161df5219538eb003e4b5343f989aaa00f96321559063ed8c8f29fd2
 # https://libisl.sourceforge.io/isl-0.26.tar.xz
 SHA512 (isl-0.26.tar.xz) = 9b5ec16d14e48f9ac9bf9cd379d3022959cfc617ade9e0d4caf2862299564fecba09d67dbdf1a4071f2f743a4fd0fabd0b0c3d15f5cddfe7226cdd5d6c2a0c66
-# https://musl.libc.org/releases/musl-1.2.4.tar.gz
-SHA512 (musl-1.2.4.tar.gz) = 498ec5d7941194a8806f4d42f0f6d218c862996ef1398b737d0d06995e0b7a6574b240a48088f6b84016b14b2776fe463f829dcb11149cdfc1023d496b235c55
-# https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.10.228.tar.xz
-SHA512 (linux-5.10.228.tar.xz) = a6d60f086c5bca2b5b18e5ec3fd410198edf4a34c8f6d3f4f9ba39a1346db9289647a9ea58eb44b3f73f2cf39aef97bd5364876b0a28619cbf5e478ec2e9977f
+# https://musl.libc.org/releases/musl-1.2.5.tar.gz
+SHA512 (musl-1.2.5.tar.gz) = 7bb7f7833923cd69c7a1a9b8a5f1784bfd5289663eb6061dcd43d583e45987df8a68a1be05d75cc1c88a3f5b610653d1a70f4a9cff4d8f7fd41ae73ee058c17c
+# https://cdn.kernel.org/pub/linux/kernel/v5.x/linux-5.10.232.tar.xz
+SHA512 (linux-5.10.232.tar.xz) = d0f3c2c6f9e47ed8e5de8e99c3e7f3175b6cb015916131ea9f7a9f1ee8aa29c7ae2fa432bcd1283bd0529654874f4f67ace208d2a8d01fc6595e4abe36ff1fd0
 # https://mirrors.kernel.org/gnu/m4/m4-1.4.19.tar.xz
 SHA512 (m4-1.4.19.tar.xz) = 47f595845c89709727bda0b3fc78e3188ef78ec818965b395532e7041cabe9e49677ee4aca3d042930095a7f8df81de3da1026b23b6897be471f6cf13ddd512b
-# https://mirrors.kernel.org/gnu/mpc/mpc-1.2.1.tar.gz
-SHA512 (mpc-1.2.1.tar.gz) = 3279f813ab37f47fdcc800e4ac5f306417d07f539593ca715876e43e04896e1d5bceccfb288ef2908a3f24b760747d0dbd0392a24b9b341bc3e12082e5c836ee
+# https://mirrors.kernel.org/gnu/mpc/mpc-1.3.1.tar.gz
+SHA512 (mpc-1.3.1.tar.gz) = 4bab4ef6076f8c5dfdc99d810b51108ced61ea2942ba0c1c932d624360a5473df20d32b300fc76f2ba4aa2a97e1f275c9fd494a1ba9f07c4cb2ad7ceaeb1ae97
 # https://mirrors.kernel.org/gnu/mpfr/mpfr-4.1.1.tar.xz
 SHA512 (mpfr-4.1.1.tar.xz) = be468749bd88870dec37be35e544983a8fb7bda638eb9414c37334b9d553099ea2aa067045f51ae2c8ab86d852ef833e18161d173e414af0928e9a438c9b91f1
-# http://cvsweb.netbsd.org/bsdweb.cgi/src/sys/sys/queue.h?rev=1.70
+# https://raw.githubusercontent.com/NetBSD/src/03be82a6b173b3c62116b7a186067fed3004dd44/sys/sys/queue.h
 SHA512 (queue.h) = 2f0d5e6e4dc3350285cf17009265dddcbe12431c111868eea39bc8cb038ab7c1f2acacbb21735c4e9d4a1fd106a8fc0f8611ea33987d4faba37dde5ce6da0750
+# https://mirrors.kernel.org/gnu/tar/tar-1.35.cpio.gz
```

Comment:
Is this going to be automatically build when build-root is compiled?

Reply:
> Is this going to be automatically build when build-root is compiled?

Yes, these are all host programs that Buildroot builds before it compiles the target toolchain.